### PR TITLE
Tennis environments

### PR DESCRIPTION
**Pull request by OryJonay**

### CODE REVIEW

File: oddsgym/envs/tennis.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    def __init__(self, tennis_bets_dataframe):
+        """Initializes a new environment.
+
+        Parameters
+        ----------
+        tennis_bets_dataframe: dataframe of shape (n_matches, n_odds + 3), n_odds == 2
+            A list of matches, with their betting odds, results and the players playing.
+
+            .. warning::
```
Comment (dgingi) [inline oddsgym/envs/tennis.py:18]: what happens when it's not this way? would the init fail?

File: oddsgym/envs/tennisdata.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import itertools
+import os
+import numpy
+import pandas
+import gym
+from .tennis import TennisOddsEnv
+from ..utils.constants.tennis import CSV_CACHE_PATH, CSV_URL, TOURNAMENTS, SITES, YEARS
+
+
+class TennisDataMixin(object):
```
Comment (dgingi) [inline oddsgym/envs/tennisdata.py:10]: should be in the same class or as a separated one?

**Review by dgingi — COMMENTED** [review]
LGTM 👍 👍 👍

File: oddsgym/envs/tennis.py (hunk 1 above)
Comment (OryJonay) [reply oddsgym/envs/tennis.py:18]: Yes, as desiged

File: oddsgym/envs/tennisdata.py (hunk 2 above)
Comment (OryJonay) [reply oddsgym/envs/tennisdata.py:10]: I want to rethink the whole design here; though about doing it in this branch, but it looked like a big detour.
We can redesign how we build those classes in a separate branch (like the metaprogramming one).

**Pull request merged**